tests/end2end/playwright/popup.spec.js

```diff
@@ -246,184 +246,189 @@ test.describe('Raster identify',
         });
-        await page.waitForTimeout(300);
-        await page.getByRole('button', { name: 'tab2' }).click({ force: true });
-        await expect(page.locator('#popup_dd_1_tab2')).toHaveClass(/active/);
-    });
 
-    test('displays children popups', async ({ page }) => {
-        await page.locator('#newOlMap').click({
-            position: {
-                x: 510,
-                y: 415
-            }
+        test('changes popup tab', async ({ page }) => {
+            const project = new ProjectPage(page, 'popup');
+            await project.open();
+
+            let getFeatureInfoPromise = project.waitForGetFeatureInfoRequest();
+
+            // When clicking `tab2`, `tab2_value` must appear
+            await project.clickOnMap(510, 490);
+            await getFeatureInfoPromise;
+            await page.getByRole('button', { name: 'tab2' }).click({ force: true });
+            await expect(page.locator('#popup_dd_1_tab2')).toHaveClass(/active/);
         });
-        await expect(page.locator('#newOlMap #liz_layer_popup .lizmapPopupChildren .lizmapPopupSingleFeature')).toHaveCount(2);
-    });
 
-    test('getFeatureInfo request should contain a FILTERTOKEN parameter when the layer is filtered', async ({ page }) => {
-        await page.locator('#button-filter').click();
+        test('displays children popups', async ({ page }) => {
+            const project = new ProjectPage(page, 'popup');
+            await project.open();
 
-        // Select a feature to filter and wait for GetMap request with FILTERTOKEN parameter
-        let getMapRequestPromise = page.waitForRequest(/FILTERTOKEN/);
-        await page.locator('#liz-filter-field-test').selectOption('1');
-        await getMapRequestPromise;
+            let getFeatureInfoPromise = project.waitForGetFeatureInfoRequest();
 
-        let getFeatureInfoRequestPromise = page.waitForRequest(request => request.method() === 'POST' && request.postData()?.includes('GetFeatureInfo') === true);
-        await page.locator('#newOlMap').click({
-            position: {
-                x: 486,
-                y: 136
-            }
+            await project.clickOnMap(510, 415);
+            await getFeatureInfoPromise;
+            await expect(page.locator('#newOlMap #liz_layer_popup .lizmapPopupChildren .lizmapPopupSingleFeature')).toHaveCount(2);
         });
 
-        let getFeatureInfoRequest = await getFeatureInfoRequestPromise;
-        expect(getFeatureInfoRequest.postData()).toMatch(/FILTERTOKEN/);
-    });
+        test('displays children in lizmap-features-table', async ({ page }) => {
+            await page.route('**/service/getProjectConfig*', async route => {
+                const response = await route.fetch();
+                const json = await response.json();
+                json.layers.dnd_popup['children_lizmap_features_table'] = 'True';
```

Comment:
```suggestion
                json.layers.dnd_popup['children_lizmap_features_table'] = true;
```

Just to confirm, `children_lizmap_features_table` will be a "true" boolean value, not a string.